endpoints/info/bidders.go

```diff
@@ -13,9 +13,10 @@ import (
 
 // NewBiddersEndpoint implements /info/bidders
 func NewBiddersEndpoint(aliases map[string]string) httprouter.Handle {
-	bidderNames := make([]string, 0, len(openrtb_ext.BidderMap)+len(aliases))
-	for bidderName := range openrtb_ext.BidderMap {
-		bidderNames = append(bidderNames, bidderName)
+	bidderNames := make([]string, 0)
```

Comment:
Should we do `bidderNames := make([]string, 0, len(openrtb_ext.CoreBidderNames()) + len(aliases))`? Perhaps also save the list from `CoreBidderNames()` so we don't have to call it twice?

Reply:
I could. This part of the code only runs once on startup. I wasn't sure if avoiding a few small memory reallocations was worth it. If you feel strongly about this, I can refactor.

Reply:
I am not too worried, but we do make a call to `CoreBidderNames()` below so we could reuse the call. I would not have said anything except that the old code had the optimization, so seems a step backwards.